modules/models.py

```diff
@@ -404,6 +405,14 @@ def ExLlamav2_HF_loader(model_name):
 
     return Exllamav2HF.from_pretrained(model_name)
 
+def HQQ_loader(model_name):
+    from hqq.engine.hf import HQQModelForCausalLM, AutoTokenizer
+
+    model_dir = f'{shared.args.model_dir}/{model_name}'
+    logger.warning(f"loading HQQ model from {model_dir}")
+    model = HQQModelForCausalLM.from_quantized(model_dir)
+    tokenizer = AutoTokenizer.from_pretrained(model_dir)
```

Comment:
You could probably just return model and let ooba handle the tokenizer options like AWQ does.

Reply:
good call. changed it

Reply:
Thanks, looks good.